sklearn/feature_extraction/_dict_vectorizer.py

```diff
@@ -384,6 +385,7 @@ def get_feature_names_out(self, input_features=None):
         feature_names_out : ndarray of str objects
             Transformed feature names.
         """
+        check_is_fitted(self, "n_features_in_")
```

Comment:
This transformer does not have a `n_features_in_` fitted attribute. Since below, we need `feature_names_`, I think that we can adapt the code with:

```suggestion
        check_is_fitted(self, "feature_names_")
```

Reply:
This should solve the failures that we see in the CIs.